contracts/BorValidatorSet.template

```diff
@@ -20,12 +20,7 @@ contract BorValidatorSet is System, ValidatorSet {
   bytes32 public constant BOR_ID = keccak256("{{borChainId}}");
   uint8 public constant VOTE_TYPE = 2;
   uint256 public constant FIRST_END_BLOCK = {{firstEndBlock}};
-
-  // sprint
-  uint256 public sprint;
-
-  // pending span proposal
-  bool private _isSpanProposalPending;
+  uint256 public constant sprint = 64;
```

Comment:
Also use `sprint` in `currentSprint` method.

Reply:
Done.